### ISSUE TITLE
Is this compatible with github's account switching?

### ISSUE THREAD

**Issue by ittayd** [yuezk/github-account-switcher#12]
Github supports account account switching natively. So this extension needs only to switch by rules. 

![image](https://github.com/yuezk/github-account-switcher/assets/33328/e4e3e33c-0642-49d0-be24-bb5c30906b89)

**Comment by yuezk** [comment]
Yes. The native switching is good. You may no longer need this extension. I will evaluate if it is necessary to support switching GitHub accounts by rules.
